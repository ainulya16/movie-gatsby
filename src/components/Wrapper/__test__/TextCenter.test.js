import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import TextCenter from '../TextCenter';

describe('TextCenter', () => {
  it('should be exists', () => {
    const wrapper = shallow(<TextCenter />);
    expect(wrapper.exists()).toBe(true);
  });
  it('should match the snapshot', () => {
    const tree = renderer.create(<TextCenter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
