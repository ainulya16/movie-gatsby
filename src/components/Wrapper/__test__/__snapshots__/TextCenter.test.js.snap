// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextCenter should match the snapshot 1`] = `
.c0 {
  text-align: center;
}

<div
  className="c0"
/>
`;
